checks/evaluation/vulnerabilities.go

```diff
@@ -31,20 +32,25 @@ func Vulnerabilities(name string, dl checker.DetailLogger,
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	score := checker.MaxResultScore
-	IDs := []string{}
+	aliasVulnerabilities := []grouper.IDAliases{}
 	for _, vuln := range r.Vulnerabilities {
-		IDs = append(IDs, vuln.ID)
-		score--
+		aliasVulnerabilities = append(aliasVulnerabilities, grouper.IDAliases(vuln))
```

Comment:
Out of curiosity, what does the `grouper` package do? Is it generic enough to be used for non-OSV usecases ?

Reply:
As long as a list of vulnerabilities have ID fields and Alias fields, grouper will be able to group vulnerabilities together so that the same vulnerability published by multiple advisories won't be counted multiple times.

More details on the alias field here: https://ossf.github.io/osv-schema/#aliases-field

Reply:
If they don't support it, it aliases field would be empty, and `IDAliases()` would return the same list. Correct?

Reply:
Yep